README.md

```diff
@@ -244,6 +259,7 @@ Options:
       --version                  Display version information
   -b, --build                    Starts the build process
       --enable-gpl-and-non-free  Enable non-free codecs  - https://ffmpeg.org/legal.html
+      --latest                   Build latest version of dependencies if newer available
```

Comment:
Should we also add `--latest` to this step in jelly-engine? https://github.com/Footage-Firm/jelly-engine/blob/master/Makefile#L34